src/azure-cli/azure/cli/command_modules/cosmosdb/aaz/latest/cosmosdb/postgres/cluster/_create.py

```diff
@@ -0,0 +1,595 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cosmosdb postgres cluster create",
+)
+class Create(AAZCommand):
+    """Create a new cluster with servers.
+    """
```

Comment:
Please add examples in aaz-dev-tools. Here's the doc [link](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#command-examples)

Reply:
Added examples